test/api/v01/vrp_test.rb

```diff
@@ -20,9 +20,14 @@
 
 class Api::V01::VrpTest < Minitest::Test
   include Rack::Test::Methods
-  include FakeRedis
   include TestHelper
 
+  def clear_optim_redis_count
+    OptimizerWrapper.config[:redis_count].keys.select{ |key| key =~ /optimizer:/ }.each do |to_remove|
```

Comment:
Where does the `optimizer` comes from ?

Reply:
`optimizer` is the prefix of all redis count keys (see other opened thread)